src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/LicenceStatisticsController.kt

```diff
@@ -18,6 +18,7 @@ import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model.LicenceStatis
 import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.LicenceStatisticsService
 import java.time.LocalDate
 
+@Deprecated(message = "Never used and likely to be removed")
```

Comment:
executive decision - I've deprecated this. We don't have plans to add this in and most of our stats are done by PA or should be done via aggregation and AP